### ISSUE TITLE
The minimum time commitment for participation in the elected bodies is undefined

### ISSUE THREAD

**Issue by hober** [w3c/w3process#820]
The [TAG Charter](https://www.w3.org/2004/10/27-tag-charter.html) used to [define](https://www.w3.org/2004/10/27-tag-charter.html#:~:text=Are%20available%20to%20spend%20approximately%2025%25%20percent%20of%20their%20time%20writing%20and%20resolving%20issues.) the expectation that TAG participants

> Are available to spend approximately 25% percent of their time writing and resolving issues.

The TAG Charter has been retired and only the process document defines what the TAG is and how it operates. This requirement, or some kind of equivalent, is not in the process document, so there's no longer any defined expectation of how many hours in a week people are to make available for their TAG work.

**Comment by frivoal** [comment]
If we want to define for informative purposes what the workload is, that's a great thing to do, but the Process isn't needed for that. /Guide, a wiki maintained by the TAG, or something like that would work.

Now if we want to require that workload, and possibly put consequences for not meeting that requirement, that would belong in the Process. But I'm not convinced that's the case.

**Comment by dwsinger** [comment]
I think I disagree with @frivoal slightly. If it's documented in the process as an expectation, that makes it fair game for "ask the candidates". I'd rather not have such things off to the side in a FAQ

**Comment by martinthomson** [comment]
I'm with @dwsinger, this needs to be formalized.

I wish that it didn't, but my experience with IETF leadership appointments suggests that informal statements have a pretty serious negative effect.  The prime example is the workload and time commitment expectations of IETF area directors, which is currently a bit of a hot topic.  Informal figures range from an unrealistic 10% to an absolutely insane 100%, with most claiming something more in the 30-50% range.

The effect of an informally-developed expectation is that the number inflates when some members spend way more time than that.  When someone does that, it can shift the expectation in a way that is damaging to those who simply cannot afford that time.

I'm less concerned about people who spend less time.  In addition to the obvious need for flexibility at times, the effect of a shortfall isn't that damaging.

**Comment by cwilso** [comment]
I agree (with @hober, @dwsinger and @martinthomson ) that we should have clear expectations set.  I think these should be clearly set for all the elected positions; for the Advisory Board, we do have some detail on it in the [Being on the AB](https://www.w3.org/wiki/AB/Being_on_the_AB) page, but I don't think it's taken very seriously at nomination/election time.  Although of course we need to have flexibility at times, I think there are members (on all the bodies) who never intended to spend a large amount of time doing what needs to be done for that body, and I think a _consistent_ shortfall from (any) members (without justification - personal/professional crisis, e.g.) IS damaging to the body as a whole.  (Totally agree that some members spend way more time than average, and that has a warping effect too.) 

Although I agree with @frivoal that the Process shouldn't _need_ to lay out something that is informative, I'm concerned that it hasn't been taken seriously even as a guideline, so I think formalizing it would be a good step.
